src/content/docs/mobile-apps/quick-actions.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Quick actions 
+tags:
+  - Mobile apps
+  - New Relic mobile apps
+metaDescription: Use quick actions to display your four favorite entities. 
+---
+
+
+import mobileAppsQuickActionAndroid from 'images/mobile-apps_screenshot_full_android_quick_action.webp'
+
+import mobileAppsQuickActioniOS from 'images/mobile-apps_screenshot_full_ios_quick_action.webp'
+
+
+To keep up in the fast-paced world of engineering and observability, use quick actions to get speedy access to common features of New Relic&mdash;right on the home screen. 
+
+
+Quick actions displays up to four of your favorite entities. If you have more than four favorites, you’ll see the most recently used ones. If you haven't set up any favorites, the display shows links to the most-used capabilities (APM, alerts, synthetics, dashboards). 
+
+To use this feature, simply long press the New Relic app icon on your home page to see your shortcuts. Here are some examples:
+
+
+<SideBySide>
+  <Side>
+    **Android**
+
+        <img
+          style={{ maxWidth: '290px' }}
```

Comment:
Not sure if you can ask Rachel to give you a gif instead, but this might be a good place for a live video as opposed to a static image.

Reply:
@homelessbirds I think we can add an animated GIF or video later. Good suggestion.